app/views/purchases/edit.html.erb

```diff
@@ -25,6 +25,12 @@
 
 <section class="content">
   <div class="container-fluid">
+    <% if @audit_performed_on_purchased_items %>
+      <div class="card-header border-0 bg-gradient-warning">
+        You’ve had an audit since this purchase was started. In the case that you are correcting a typo, rather than recording that the physical amounts being (purchased/donated) have changed,
```

Comment:
```suggestion
        You’ve had an audit since this purchase was started. In the case that you are correcting a typo, rather than recording that the physical amounts being purchased have changed,
```